app/assets/stylesheets/darkswarm/style.css.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
 	font-family: 'OFN';
-	src:url('/OFN-v2.eot?eslsji');
-	src:url('/OFN-v2.eot?#iefixeslsji') format('embedded-opentype'),
-		url('/OFN-v2.woff?eslsji') format('woff'),
-		url('/OFN-v2.ttf?eslsji') format('truetype'),
-		url('/OFN-v2.svg?eslsji#OFN') format('svg');
+	src: font-url('OFN-v2.eot');
+	src: font-url('OFN-v2.eot') format('embedded-opentype'),
+		font-url('OFN-v2.woff') format('woff'),
+		font-url('OFN-v2.ttf') format('truetype'),
```

Comment:
I notice the previous versions have suffixes added, but you've removed them. Are they not needed? Eg:

`'/OFN-v2.eot?#iefixeslsji'` is now: `'OFN-v2.eot'`

Reply:
I strongly believe that was the pre-asset-pipeline poor-man's asset versioning. That's what the fringerprint came to solve way better. Unless `iefixes` refers to IE? Googling yielded no results.

Reply:
I think it might be fixes for IE, yes. I have a feeling the multiple file formats are there because they can't all be rendered in all browsers... Maybe we could have a look using Browserstack?

Reply:
:+1: I added it in the test instructions